client/components/payment-methods-list/payment-method.js

```diff
@@ -46,6 +46,7 @@ const PaymentMethod = ( {
 		return onUncheckClick( id );
 	};
 
+	//console.log( accountFees );
```

Comment:
Dead code, can we drop this, please?

Reply:
Sorry about missing this one. I have [removed it](https://github.com/Automattic/woocommerce-payments/pull/5299/commits/2d8b5afb1599fc98fa4bb1c4651818d20acc4267#diff-23a1b15bc75d0f257ab6e00340f08eaa9280872e4f83949e6e847500000491f3) 

Commit - [1ec4acd](https://github.com/Automattic/woocommerce-payments/pull/5299/commits/1ec4acd531489ad3f58d1b99b79a875f498f4862)